### PR TITLE
Supernode Integration

### PR DESCRIPTION
**Pull request by alonamid**
PR used to track changes

TODOs:
- [x] confirm xsim still works

### CODE REVIEW

**Comment by alonamid** [comment]
@sagark @davidbiancolin @zhemao This is ready for review (other than the default agfi's which I am currently generating)

File: deploy/sample-backup-configs/sample_config_build.ini [hunk 1]
```diff
@@ -21,6 +21,11 @@ firesim-quadcore-nic-ddr3-llc4mb
 fireboom-singlecore-no-nic-ddr3-llc4mb
 #fireboom-singlecore-nic-lbp
 fireboom-singlecore-nic-ddr3-llc4mb
+#firesim-supernode-singlecore-nic-ddr3-llc4mb
+#firesim-supernode-quadcore-nic-ddr3-llc4mb
+#firesim-supernode-singlecore-nic-lbp
+#firesim-supernode-quadcore-nic-lbp
```
[the remark this answers is not in the document]
Comment (sagark) [reply deploy/sample-backup-configs/sample_config_build.ini]: @alonamid did quad core ever finish building?

**Comment by sagark** [comment]
As a sanity check, I ran the memcached benchmarks on non-supernode images generated from this branch and everything looks good.

File: deploy/sample-backup-configs/sample_config_build.ini (hunk 1 above)
[the remark this answers is not in the document]
Comment (alonamid) [reply deploy/sample-backup-configs/sample_config_build.ini]: quadcore with ddr built with 75 MHz and met timing. Interestingly, quadcore with lbp seems to be having problems. Anyways, a full rebuilt with is on it's way again.

**Comment by alonamid** [comment]
Issue #189 #190 were opened

File: docs/Advanced-Usage/Supernode.rst [hunk 2]
```diff
@@ -1,32 +1,37 @@
...
-Supernode support is currently in beta. Supernode is designed to improve FPGA
-resource utilization for smaller designs and allow realistic rack topology
-simulation (32 simulated nodes) using a single ``f1.16xlarge`` instance.  The
-supernode beta can be found on the ``supernode-beta`` branch of the FireSim
-repository. Supernode requires slight changes in build and runtime
-configurations. More details about supernode can be found in the `FireSim ISCA
-2018 Paper <https://sagark.org/assets/pubs/firesim-isca2018.pdf>`__.
-
-Intro
+Supernode - Multiple Simulations Per FPGA
```
[the remark this answers is not in the document]
Comment (alonamid) [reply docs/Advanced-Usage/Supernode.rst]: I agree. I changed it the "Simulated SoCs"

**Comment by sagark** [comment]
(pending merge/bump of aws-fpga and midas)

**Pull request merged**